src/frontend/context/AuthContext.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { createContext, useEffect, type PropsWithChildren } from 'react';
+
+import { HttpProvider } from '@navikt/familie-http';
+import type { ISaksbehandler } from '@navikt/familie-typer';
+
+interface IProps extends PropsWithChildren {
```

Comment:
Drop gjerne `I` prefixen, se f.eks. [her](https://dev.to/mscamargo/why-you-should-avoid-using-the-i-prefix-for-interfaces-in-typescript-43gd#:~:text=In%20TypeScript%2C%20interfaces%20are%20often,there%20is%20only%20one%20implementation.)
 
```suggestion
interface Props extends PropsWithChildren {
```